PublishedModules/glueckself/mpdclient/mpdclient.py

```diff
@@ -0,0 +1,80 @@
+import mpdhelper
+
+from core.base.model.Intent import Intent
+from core.base.model.Module import Module
+from core.dialog.model.DialogSession import DialogSession
+from core.commons.commons import online
+
+class mpdclient(Module):
+	"""
+	Author: glueckself
+	Description: Control an MPD server
+	"""
+
+	_INTENT_PLAY = Intent('Play')
+	_INTENT_STOP = Intent('Stop')
+	_INTENT_NEXT = Intent('Next')
+	_INTENT_PREV = Intent('Prev')
+
+	def __init__(self):
+		self._SUPPORTED_INTENTS = [
+			self._INTENT_PLAY,
+			self._INTENT_STOP,
+			self._INTENT_NEXT,
+			self._INTENT_PREV
+		]
+
+		super().__init__(self._SUPPORTED_INTENTS)
+		
+		config = self.getConfig("mpdclient")
+		self._host = config.get("mpdHost")
+		self._port = config.get("mpdPort")
+		self._password = config.get("mpdPassword")
```

Comment:
single quotes please :)

Reply:
Sorry, habit, fixed.
Should I make sure that mpdhelper.py also conforms to the code guidelines?
As noted below, that is a file that I copied from another project.